starboard/win/shared/platform_configuration/BUILD.gn

```diff
@@ -42,8 +42,10 @@ config("platform_configuration") {
   ]
 
   cflags = [
-    # Add POSIX emulation headers
+    # Add POSIX emulation headers (note: sys/time.h cannot be force included)
```

Comment:
I'm thinking of separating what needs to be force included and what is in the include path and gets explicitly included by the source code. In any case what you have here is good for this PR.